### PR TITLE
back navigation icon added

### PR DESCRIPTION
**Pull request by BeakalDegefa**
## Changes
_List Changes Introduced by this PR_
-Back button added to navigate to previous screens. 

## Purpose
for navigation purpose

### CODE REVIEW

**Review by yelab07 — APPROVED** [review]
This looks good to me great job Beakal on the corrections you have made !

File: components/SearchBar.tsx [hunk 1]
```diff
@@ -1,36 +1,50 @@
...
-        }}
-        onSubmitEditing={() => {
-          navigation.navigate(
-            "Home" as never,
-            { screen: "ProductsScreen", params: { search: text } } as never
-          );
-          setText("");
-        }}
-      />
+      <View style={{ flexDirection: "row", justifyContent: "center" }}>
```
Comment (eangele1) [inline components/SearchBar.tsx:17]: Wanted to get some clarification on this part. Will this appear all the time next to the search bar or only within the products and product detail screens?

Comment (BeakalDegefa) [reply components/SearchBar.tsx:17]: It is there in all screens except the cart and check out page

Comment (BeakalDegefa) [reply components/SearchBar.tsx:17]: I corrected your suggestions.

Comment (eangele1) [reply components/SearchBar.tsx:17]: Awesome! Great work.

**Review by eangele1 — APPROVED** [review]

**Pull request merged**